src/screens/quickSurvey/QuickSurvey.js

```diff
@@ -61,34 +68,41 @@ class QuickSurvey extends Component {
     this.stateManager = undefined;
   }
 
+  /*
+    This function reads the file at a given file path. When the file is done
+    reading the data is passed to parseLoadedState and the displays a dialog
+    with a success message and the loading state is set to false. If the file
+    cannot be read, the screen will render the setup questions.
+  */
   async loadStateFromJSON() {
-    console.log("Loading State from JSON");
     await this.stateManager.RNFS.readFile(this.stateManager.path, "utf8")
       .then((data) => {
         try {
           const loadedState = JSON.parse(data);
-          console.log("Successfully Parsed JSON");
           this.parseLoadedState(loadedState);
           this.setState({ loadedAutoSaveSuccessMessageVisible: true });
         } catch (e) {
           // catch any error while parsing the JSON
           console.log("ERROR: " + e.message);
-          //                        this.setState({ loadedAutoSaveFailMessageVisible: true })
-          //            this.stateManager.deleteCapturedState();
         }
         // Hide loading screen
         this.setState({ loading: false });
       })
       .catch((err) => {
         // catch any error while reading autoSavedSession JSON from disk
-        //          this.stateManager.deleteCapturedState();
         this.setState({ autoSavedSession: false });
         this.setState({ loading: false });
-        //          this.setState({ loadedAutoSaveFailMessageVisible: true })
         console.log(err.message);
       });
   }
 
+  /*
+    This function runs during the initial render of the screen. The function
+    loads the questions from the downloaded spreadsheet and saves them to a
+    variable. If the screen is given a saved session and the data has not been
+    loaded, it loads the data from the file. Otherwise, the loading state will
+    be set to false.
```

Comment:
I don't think you need the first sentence since most devs will know when `componentDidMount` runs. This also reads more like a set of inline comments put together - I'd rephrase it to be more general, maybe something like "Loads questions from downloaded spreadsheet and instantiates a background save object in order to retrieve the in-progress report (if there was an in-progress report re-loaded)"